CHANGELOG.md

```diff
@@ -5,26 +5,26 @@ This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a ch
 
 ## [Unreleased]
 
-### Added
+### Fixed
+- *Nuget and Pypi repositories were configured only if nexus_npm_enable: true* @jnogol
 
+### Added
 - *[#7](https://github.com/idealista/nexus-role/issues/7) Support cleanup policies* @jmonterrubio
+- *[#12](https://github.com/idealista/nexus-role/issues/12) Ability to configure raw repositories* @jnogol
```

Comment:
Seems i missed 1.3.0 version in changelog :(
Could you add it?

Reply:
Done!